### PR TITLE
Improved instructions for understanding camelCase in variables challenge

### LINKED ISSUE #9254: Improve instructions for Understanding case sensitivity in variables challenge
#### Understanding Case Sensitivity in Variables

It looks like a learner had problem in understanding the instructions for [this](https://www.freecodecamp.com/challenges/understanding-case-sensitivity-in-variables) challenge.

The instructions currently read 
> Fix the variable declarations and assignments so their names use camelCase.

Maybe we can change it along the lines of  
> Modify the existing declarations and assignments so their names use camelCase. 
> In other words, do not create any new variables; just change and fix existing incorrect variable names in the declaration and assignment statements.

### PR DESCRIPTION
**Pull request by oscarkaypee**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x]  Closes currently open issue (replace XXXX with an issue no): Closes #9254 
#### Description

Removed one line that was hard to understand on the topic of camelCase (case sensitivity in variable naming) and replaced with more info to better describe the challenge.

### CODE REVIEW

**Comment by atjonathan** [comment]
@okputus, you appeared to have pulled in some commits from `master`. Try dropping commits using `git rebase -i HEAD~n` and changing all unwanted commits from `squash` to `drop`.
If you need any assistance reach out to the [Contributors Chat room](https://gitter.im/FreeCodeCamp/HelpContributors).

**Comment by okputus** [comment]
Thanks for the advice @atjonathan, have reduced the number of commits to 1 now!

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 1]
```diff
@@ -235,7 +235,7 @@
         "<strong>Examples:</strong>",
         "<blockquote>var someVariable;<br>var anotherVariableName;<br>var thisVariableNameIsTooLong;</blockquote>",
         "<h4>Instructions</h4>",
-        "Fix the variable declarations and assignments so their names use <dfn>camelCase</dfn>."
+        "Modify the existing declarations and assignments so their names use <dfn>camelCase</dfn>.<br>In other words, do not create any new variables; just fix existing incorrect variable names in the declaration and assignment statements."
```
Comment (Bouncey) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: ~~In other words, do not create any new variables; just fix existing incorrect variable names in the declaration and assignment statements~~ Do not create any new variables.

Saves from repeating ourselves.

**Comment by atjonathan** [comment]
#### :sparkles: :sparkles: **LGTM :+1:** :sparkles: :sparkles: 

:tada: Thanks for contributing @okputus and congratulations on your first PR submission to FreeCodeCamp! :tada:

**Pull request merged**